hackjohn.py

```diff
@@ -140,3 +140,27 @@ def get_trailhead_df():
     mmb_response = requests.post(mmb_url, json=payload)
     print('middleman status code', mmb_response.status_code)
     print(mmb_response.text)
+
+## Notifications using IFTTT
+
+# Set enable_ifttt to True to receive IFTTT notification
+enable_ifttt = True
```

Comment:
```suggestion
enable_ifttt = False
```

let's default to false since users will have to populate the ifttt fields first